config/config.go

```diff
@@ -0,0 +1,163 @@
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/googleinterns/aso_sxs_viewer/proto"
+	"google.golang.org/protobuf/encoding/prototext"
+)
+
+type ViewerConfig struct {
+	proto.ViewerConfig
+}
+
+type BrowserConfig struct {
```

Comment:
Like ViewerConfig, just use proto.BroswerConfig directly.

Reply:
that would require create window to interact with the generated functions like GetURL directly which I was trying to avoid. Should I do that?

Reply:
I don't see what the issue in that is?